Running eslint gives errors

Describe the bug
Running eslint returns:

/tmp/core/packages/crypto/lib/models/transaction.js
  172:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  178:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  182:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  188:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  204:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  209:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  216:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style
  224:5  error  Closing curly brace does not appear on the same line as the subsequent block  brace-style

â 8 problems (8 errors, 0 warnings)
  8 errors, 0 warnings potentially fixable with the `--fix` option.

Expected behavior
No eslint errors.

Will provide pull request for changing it.

--- a/packages/crypto/lib/models/transaction.js
+++ b/packages/crypto/lib/models/transaction.js
@@ -169,25 +169,17 @@ module.exports = class Transaction {
       bb.writeUInt64(+transaction.amount.toString())
       bb.writeUInt32(transaction.expiration || 0)
       bb.append(bs58check.decode(transaction.recipientId))
-    }
-
-    else if (transaction.type === TRANSACTION_TYPES.VOTE) {
+    } else if (transaction.type === TRANSACTION_TYPES.VOTE) {
       const voteBytes = transaction.asset.votes.map(vote => (vote[0] === '+' ? '01' : '00') + vote.slice(1)).join('')
       bb.writeByte(transaction.asset.votes.length)
       bb.append(voteBytes, 'hex')
-    }
-
-    else if (transaction.type === TRANSACTION_TYPES.SECOND_SIGNATURE) {
+    } else if (transaction.type === TRANSACTION_TYPES.SECOND_SIGNATURE) {
       bb.append(transaction.asset.signature.publicKey, 'hex')
-    }
-
-    else if (transaction.type === TRANSACTION_TYPES.DELEGATE_REGISTRATION) {
+    } else if (transaction.type === TRANSACTION_TYPES.DELEGATE_REGISTRATION) {
       const delegateBytes = Buffer.from(transaction.asset.delegate.username, 'utf8')
       bb.writeByte(delegateBytes.length)
       bb.append(delegateBytes, 'hex')
-    }
-
-    else if (transaction.type === TRANSACTION_TYPES.MULTI_SIGNATURE) {
+    } else if (transaction.type === TRANSACTION_TYPES.MULTI_SIGNATURE) {
       let joined = null
 
       if (!transaction.version || transaction.version === 1) {
@@ -201,29 +193,21 @@ module.exports = class Transaction {
       bb.writeByte(transaction.asset.multisignature.keysgroup.length)
       bb.writeByte(transaction.asset.multisignature.lifetime)
       bb.append(keysgroupBuffer, 'hex')
-    }
-
-    else if (transaction.type === TRANSACTION_TYPES.IPFS) {
+    } else if (transaction.type === TRANSACTION_TYPES.IPFS) {
       bb.writeByte(transaction.asset.ipfs.dag.length / 2)
       bb.append(transaction.asset.ipfs.dag, 'hex')
-    }
-
-    else if (transaction.type === TRANSACTION_TYPES.TIMELOCK_TRANSFER) {
+    } else if (transaction.type === TRANSACTION_TYPES.TIMELOCK_TRANSFER) {
       bb.writeUInt64(+transaction.amount.toString())
       bb.writeByte(transaction.timelockType)
       bb.writeUInt32(transaction.timelock)
       bb.append(bs58check.decode(transaction.recipientId))
-    }
-
-    else if (transaction.type === TRANSACTION_TYPES.MULTI_PAYMENT) {
+    } else if (transaction.type === TRANSACTION_TYPES.MULTI_PAYMENT) {
       bb.writeUInt32(transaction.asset.payments.length)
       transaction.asset.payments.forEach(p => {
         bb.writeUInt64(p.amount)
         bb.append(bs58check.decode(p.recipientId))
       })
-    }
-
-    else if (transaction.type === TRANSACTION_TYPES.DELEGATE_RESIGNATION) {
+    } else if (transaction.type === TRANSACTION_TYPES.DELEGATE_RESIGNATION) {
        // delegate resignation - empty payload
     }
 

